### DISCUSSION TITLE
Are the dates on the blog correct?

### DISCUSSION THREAD

**Discussion by dbarkelew** [FreeTubeApp/FreeTube#5377] [Q&A]
I just updated the app, Then the banner that typically alerts me to updates alerted for a new blog post. However, the post date is from 2021. Is the banner old, or is the date on the blog post wrong?

This is the link https://blog.freetubeapp.io/behind-the-scenes-improving-project-structure-and-other-updates?pk_campaign=rss-feed

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
Hi @dbarkelew the dates on the blog post are correct. Is this the first time you installed FreeTube?

**Reply by dbarkelew** [reply]
Pretty much, and finally using it a bit more now. The search and organization is way better, and I love my searches now have results. Not trending recommendations that have nothing to do with my search.

**Reply by efb4f5ff-1298-471a-8973-3d47447115dc** [reply]
Ah okay, well that makes allot of sense. FreeTube checks at startup if there has been a new blog post since the last time you opened FreeTube. 

If you open FreeTube for the first time it will always say that there is a "new" blog post available
